lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -78,91 +77,39 @@ defmodule PhxToolsWeb.PhxToolsComponents do
               </div>
-              <p class="text-xs leading-5 md:leading-6 md:text-sm lg:text-base text-center sm:px-3 font-martian sm:py-6 md:px-8">
-                It looks like you're using an operating system that Phx.tools doesn't currently support. This script is designed to work on Linux and macOS only. Please switch to a compatible operating system to continue.
-              </p>
             </div>
           </div>
-          <.web_updates />
-          <.footer />
-        </div>
-      </div>
-    </div>
-    """
-  end
-
-  def os_landing_card(assigns) do
-    ~H"""
-    <div class="solved-height">
-      <div class="min-h-full">
-        <div class="mt-[5%] md:flex flex-col items-center max-w-4xl mx-auto">
-          <div class="sm:flex flex-col items-center">
-            <Icons.phx_tools_svg class="h-7 md:h-12" />
-            <p class="sm:py-6 text-center text-white text-sm md:text-base lg:text-lg font-martian font-semibold ">
-              The Complete Development Environment for Elixir and Phoenix
-            </p>
-          </div>
-
-          <div class="md:flex lg:w-[75%] flex-col relative">
-            <div class="md:flex">
-              <div class="flex justify-end items-center md:order-last md:mx-2 sm:mb-2">
-                <Heroicons.information_circle
-                  class="hidden md:block w-6 h-6 text-white cursor-pointer focus:text-gray-500"
-                  phx-click={JS.show(to: "#installation-instructions")}
-                />
-              </div>
-              <div class="border sm:px-3 bg-[#26168780] w-full flex items-center justify-center py-2 sm:space-x-3 rounded-sm overflow-x-auto whitespace-nowrap no-scrollbar">
-                <h1 id="tool-installation" class="text-center text-white font-martian sm:text-sm">
-                  <%= render_slot(@installation_command) %>
-                </h1>
-                <div id="copy" phx-hook="CopyHook" class="sm:py-3 md:my-0 cursor-pointer">
-                  <Icons.copied_icon />
-                  <Icons.copy_icon />
-                </div>
+          <div class=" lg:w-[1100px]">
+            <div class=" w-[100%] flex flex-col gap-8 items-center justify-center">
+              <div class="md:w-[65%]">
+                <p class="text-xs px-2 leading-5 text-center text-white font-martian  md:text-sm md:leading-6 lg:text-base">
+                  <%= phx_tools_description() %>
+                </p>
               </div>
-            </div>
-            <div class="w-full bg-blue-600 flex justify-center">
-              <.os_instructions
-                :let={installation_instruction}
-                installation_instructions={installation_instructions(@live_action)}
-                live_action={@live_action}
-                operating_system={@operating_system}
+              <div
+                :if={assigns.live_action == :index}
```

Comment:
We have only one page, so no need for this check.

Reply:
![Screenshot from 2024-11-22 09-35-53](https://github.com/user-attachments/assets/5922762b-9657-4498-93b8-c8cadc7cffa5)
From this we need that check to determine whether or not to display the unsupported os message

Reply:
Yeah, but this can be even in the `mount` function, if I'm not mistaken.